src/pycram/description.py

```diff
@@ -360,16 +425,24 @@ def get_origin_transform(self) -> Transform:
     @property
     def color(self) -> Color:
         """
-        The getter for the rgba_color of this link.
+        The rgba_color of this link.
 
         :return: A Color object containing the rgba_color of this link.
         """
         return self.world.get_link_color(self)
 
+    def set_color(self, color: Color) -> None:
+        """
+        Set the color of this link, could be rgb or rgba.
+
```

Comment:
why does this exist?

Reply:
for old uses, added a deprecation warning to it